roles/confluent.test/molecule/zookeeper-mtls/verify.yml

```diff
@@ -1,4 +1,11 @@
 ---
+- name: Verify - Confluent CLI
```

Comment:
also add this to the `archive-community-plaintext-rhel/verify.yml` file to prove that one worked as well. looks perfect

Reply:
added